image/cli/mascli/functions/mirror_to_registry

```diff
@@ -550,24 +527,17 @@ function mirror_to_registry() {
     if [[ "$MIRROR_MAS_IOT" == "true" || "$MIRROR_MAS_MANAGE" == "true" ]]; then
       show_mirror_status " + IBM Db2 .........................." $MIRROR_DB2U
     fi
-    # Assist, HPU & Predict Dependencies
-    if [[ "$MIRROR_MAS_ASSIST" == "true" || "$MIRROR_MAS_HPUTILITIES" == "true" || "$MIRROR_MAS_PREDICT" == "true" ]]; then
+    # Assist & Predict Dependencies
+    if [[ "$MIRROR_MAS_ASSIST" == "true" || "$MIRROR_MAS_PREDICT" == "true" ]]; then
       show_mirror_status " + IBM Cloud Pak for Data (CP4D) ...." $MIRROR_CP4D
     fi
-    # HPU & Predict Dependencies
-    if [[ "$MIRROR_MAS_HPUTILITIES" == "true" || "$MIRROR_MAS_PREDICT" == "true" ]]; then
-      show_mirror_status " + IBM Watson Studio ................" $MIRROR_WSL
```

Comment:
Do we not need WSL for Predict?

Reply:
See line 540 in the changed file ... this now shows up in the Predict specific section